src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/prisonperson/identifyingmarks/IdentifyingMarkResource.kt

```diff
@@ -87,5 +89,41 @@ class IdentifyingMarkResource(private val service: IdentifyingMarkService) {
   )
   suspend fun getIdentifyingMarkMappings(
     @PathVariable dpsId: UUID,
-  ) = service.getIdentifyingMarkMappings(dpsId).map { it.toDto() }
+  ) = service.getIdentifyingMarkMappings(dpsId)
+
+  @PostMapping("/identifying-mark")
```

Comment:
The main change for this PR is to create new endpoint to create identifying marks mappings